app/cypress/e2e/SecvisogramPage.cy.js

```diff
@@ -2,8 +2,8 @@
 
 import CVSSVector from '../../lib/app/SecvisogramPage/View/FormEditor/editors/GenericEditor/Attributes/CVSS3Attribute/CVSSVector.js'
 import ViewReducer from '../../lib/app/SecvisogramPage/View/Reducer.js'
-import docMax from '../../lib/app/shared/Core/doc-max.json'
-import docMin from '../../lib/app/shared/Core/doc-min.json'
+import docMax from '../../lib/core/v2_1/doc-max.json'
+import docMin from '../../lib/core/v2_1/doc-min.json'
```

Comment:
I think the templates here should still be from version 2.0 since the tests still operate in this mode.

Reply:
With this, the test 'can validate the document against the rest service' fails.
Where can I see that the tests start with version 2.0.
The application starts with 2.1.